### PR TITLE
feat: update MintedImg component

### PR DESCRIPTION
**Pull request by zhangjing28**
## Description

* update the MintedImg accroding to [new design](https://www.figma.com/file/t7AdCuflV4kJhb9H3O3Zjz/NPO-SBT?node-id=52%3A11688&t=oHw8baoOWOdYANA6-0)
* change "zkSBT ID" to "Proof Key"
* update the copy icon according to the new design

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [X] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -60,7 +60,8 @@ module.exports = {
         'connect-wallet-button': '#00AFA5',
         'gradient-button': 'var(--gradient-button-secondary)',
         'light-warning': '#FF8A0020',
-        'light-check': '#29CCB950'
+        'light-check': '#29CCB950',
+        cover: 'var(--color-bg-cover)'
```
Comment (jincan39) [inline tailwind.config.js:64]: suggest the usage of 'bg-black bg-opacity-60' to cater for tailwind's land.

**Review by jincan39 — APPROVED** [review]
LGTM

File: tailwind.config.js (hunk 1 above)
Comment (zhangjing28) [reply tailwind.config.js:64]: Good to know that, will do it

**Review by jincan39 — APPROVED** [review]
LGTM

**Pull request merged**